### DISCUSSION TITLE
Bug well know from matplotlib 3.5.1

### DISCUSSION THREAD

**Discussion by rbregand** [brainvisa/casa-distro#342] [General]
Hi, 

I just noticed a known issue with matplotlib 3.5.1 which is the version installed into casa-dev-5.3.19 image.
https://github.com/matplotlib/matplotlib/issues/21688

To dodge this issue, I use the `pip3 install --user matplotlib==3.4.3` command as recommended in the previous link.
I don't know if you plan to change the matplotlib version, I just wanted you to be aware of this bug.

Rémy

**Comment by sapetnioc** [comment]
Matplotlib was installed with `apt` so we cannot select the version this way. I made a (temporary) change to install it with `pip` and forced the version to `3.4.3`. I rebuilt and published [casa-run-5.3-18](https://brainvisa.info/download/casa-run-5.3-18.sif) and [casa-dev-5.3-20](https://brainvisa.info/download/casa-dev-5.3-20.sif) images.

**Comment by rbregand** [comment]
Ok thanks !
